src/utils/lodash/index.ts

```diff
@@ -0,0 +1,7 @@
+import debounce from 'lodash/debounce';
```

Comment:
@ArmandoGraterol Do you think this is a reasonable compromise? Only importing the functions we use but still getting them into the relevant file on one line.

Reply:
I think it's a great idea